src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -400,80 +370,45 @@ public void buttonDeleteCvrFileClicked() {
   }
 
   public void changeCvrFilePath(CellEditEvent cellEditEvent) {
-    CVRSource cvrSelected = tableViewCvrFiles.getSelectionModel().getSelectedItem();
-    // Cache the original value to revert back to in case the change fails
-    String oldFilePath = cvrSelected.getFilePath();
-    cvrSelected.setFilePath(cellEditEvent.getNewValue().toString().trim());
-    if (!cvrSourceHasRequiredFields(cvrSelected)) {
-      cvrSelected.setFilePath(oldFilePath);
-    }
+    tableViewCvrFiles.getSelectionModel().getSelectedItem()
```

Comment:
Could you explain what these changes do?

Reply:
Per the commit message: "disables validation check when changing cells to avoid situation where config with bad sources is loaded in the GUI and can't get corrected." So if I create a bad entry directly in the JSON with multiple bad fields, this allows you to actually fix them all. Otherwise, it resets to the bad fields across the board.